backend/api/__init__.py

```diff
@@ -64,11 +64,13 @@ def create_app(test_config=None):
     from api.views import authenticate
     from api.views import quiz
     from api.views import book
+    from api.views import user
 
     app.register_blueprint(main.main)
     app.register_blueprint(authenticate.authenticate)
     app.register_blueprint(quiz.quiz)
     app.register_blueprint(book.book)
+    app.register_blueprint(user.user)
```

Comment:
nice job :star: with separating endpoints, i should've done it earlier with GET /user